Eevee.Sleep.Bot/Controllers/Mongo/MongoConst.cs

```diff
@@ -20,4 +20,12 @@ public static class MongoConst {
 
     public static readonly IMongoCollection<ActivationPresetModel> AuthActivationPresetCollection =
         AuthDatabase.GetCollection<ActivationPresetModel>("activationPreset");
+
+    private static readonly IMongoDatabase DiscordDatabase = Client.GetDatabase("discord");
+
+    public static readonly IMongoCollection<DiscordRoleModel> RoleCollection =
```

Comment:
Name it `role/record` and `role/tracked`.

Since the DB of `discord` could get used or other things, this naming helps categorizing the collections so we know that these 2 are doing role-related stuff.